elasticsearch/README.md

```diff
@@ -42,16 +42,13 @@ For further configuration see below:
 ### Defaults Configuration
 The default setting for the Elasticsearch node that is created is as follows:
 
-    cluster.name=es.ycsb.cluster
-    node.local=true
-    path.data=$TEMP_DIR/esdata
-    discovery.zen.ping.multicast.enabled=false
-    index.mapping._id.indexed=true
-    index.gateway.type=none
-    gateway.type=none
-    index.number_of_shards=1
-    index.number_of_replicas=0
+    es.cluster.name=es.ycsb.cluster
```

Comment:
release manager that covers when this gets published should call this out as an incompatible change.